src/main/scala/com/twitter/finagle/websocket/WebSocketHandler.scala

```diff
@@ -10,11 +9,38 @@ import org.jboss.netty.handler.codec.http.websocketx._
 import org.jboss.netty.handler.codec.http.{HttpHeaders, HttpRequest, HttpResponse}
 import scala.collection.JavaConversions._
 
+private[finagle] class CancelledException extends Exception
+
 class WebSocketHandler extends SimpleChannelHandler {
   protected[this] val messagesBroker = new Broker[String]
   protected[this] val binaryMessagesBroker = new Broker[Array[Byte]]
   protected[this] val closer = new Promise[Unit]
 
+  implicit class RichChannelFuture(future: ChannelFuture) {
+    private[this] val promise = new Promise[Unit]
+
+    future.addListener(new ChannelFutureListener {
+      def operationComplete(cf: ChannelFuture) {
+        if(cf.isSuccess)
+          promise.setValue(())
+        else if(cf.isCancelled)
+          promise.setException(new CancelledException)
+        else
+          promise.setException(cf.getCause)
+      }
+    })
+
+
+    def toFuture:Future[Unit] = {
```

Comment:
This is super nitpicky, but can we have a space after the colon and before the type for the sake of consistency?